docs/yaml/builtins/meson.yaml

```
@@ -164,6 +164,21 @@ methods:
       - `vs2022`
       - `xcode`
 
+  - name: build_options
+    returns: str
+    since: 1.1.0
+    description: |
+      Returns a string with the configuration line used to set the current project up.
+    notes:
+      - |
+        **Do not try to parse this string!**
+
+        You should use [[confdata.set_quoted]] to safely escape any embedded
```

```suggestion
        You should use [[cfg_data.set_quoted]] to safely escape any embedded
```

This comes from docs/yaml/objects/cfg_data.yaml as `name: cfg_data`.